01-JS-DOM-APIs/9/scripts/request.js

```diff
@@ -0,0 +1,40 @@
+function createRepoList(items) {
+	const sectionList = document.getElementById("sectionList");
+	const oldList = document.getElementById("list");
+	const newList = document.createElement("ul");
+	
+	for (var i = 0; i < items.length; i++) {
```

Comment:
All var declarations should be at the top of the function even for iterator `i`. Search about **hoisting**.